packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -4,6 +4,8 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import { SfProject } from '@salesforce/core';
```

Comment:
It's be nice if we could use a function in `salesforcedx-utils-vscode` to do the verification and not add a new `@salesforce/core` import in the `salesforcedx-vscode-core` package.